cypress/integration/models/developer/applicationinventory/analysis.ts

```diff
@@ -167,11 +170,25 @@ export class Analysis extends Application {
         }
     }
 
+    validateAvailableActions(toBePresent: boolean): void {
```

Comment:
This is the correct place for this method :+1: 
In analysis model .

Reply:
I will do this in next PR, I would like to close this one already, is this OK?